test/parallel/test-process-hrtime.js

```diff
@@ -24,3 +24,9 @@ function validateTuple(tuple) {
     assert(isFinite(v));
   });
 }
+
+const base = process.hrtime();
+for (let i = 0; i < 1e5; i += 1) {
+  const diff = process.hrtime(base);
+  assert(diff[1] >= 0);  // https://github.com/nodejs/node/issues/4751
+}
```

Comment:
The current testcase that you added will either slow down the tests by more than one second or will have a significant chance of false negatives, depending on the `for` cycle limit and the machine speed.